blocks/bricks/__init__.py

```diff
@@ -658,10 +658,17 @@ def add_property(f):
         return add_property
 
     def __getattr__(self, attr):
+        # Prevents infinite recursion when no attribute "f" is available
+        # (happens during serialization under unknown circumstances)
+        try:
+            self_f = self.__getattribute__("f")
```

Comment:
But if the `f` value is available, `__getattr__` would never be called. `__getattr__` is only called if the attribute isn't available.